src/components/templates/Auditorium/components/AllSectionPreviews/AllSectionPreviews.tsx

```diff
@@ -62,50 +60,51 @@ export const AllSectionPreviews: React.FC<SectionPreviewsProps> = ({
   }
 
   return (
-    <>
-      <VenueWithOverlay venue={venue} containerClassNames="">
-        <InfiniteScroll
-          dataLength={auditoriumSections.length}
-          className={`AllSectionPreviews ${containerClasses}`}
-          next={loadMore}
-          hasMore={
-            venue.sectionsCount
-              ? auditoriumSections.length < venue.sectionsCount
-              : true
-          }
-          loader={<Loading containerClassName="AllSectionPreviews__loader" />}
+    <VenueWithOverlay venue={venue} containerClassNames="">
+      <InfiniteScroll
+        dataLength={auditoriumSections.length}
+        className={`AllSectionPreviews ${containerClasses}`}
+        next={loadMore}
+        hasMore={
+          venue.sectionsCount
+            ? auditoriumSections.length < venue.sectionsCount
+            : true
+        }
```

Comment:
[NOTE]
```suggestion
        hasMore={!venue.sectionsCount || auditoriumSections.length < venue.sectionsCount}
```

Reply:
This is harder to follow with `||` and `<` priorities 🤔 OK, I'll accept it